hs_core/views/__init__.py

```diff
@@ -380,7 +380,7 @@ class FilterForm(forms.Form):
     start = forms.IntegerField(required=False)
     published = forms.BooleanField(required=False)
     edit_permission = forms.BooleanField(required=False)
-    creator = forms.ModelChoiceField(queryset=User.objects.all(), required=False)
```

Comment:
I didn't realize this search form was used outside the search page. That's a whole lot of places coupled to this one form. Do all of those examples actually user the FilterForm class? I don't see where some of them are doing so at all. Could you explain?